dockerscripts/docker-entrypoint.sh

```diff
@@ -37,7 +37,75 @@ docker_secrets_env() {
     fi
 }
 
+## Create UID/GID based on available environment variables.
+docker_set_uid_gid() {
+    if  [ -z "$MINIO_GID" ] ||  [ -z "$MINIO_UID" ] ; then
+	    addgroup -g 190 -S minio && adduser -u 190 -S -G minio minio
+    else
+        addgroup -g "$MINIO_GID" -S minio && adduser -u "$MINIO_UID" -S -G minio minio
```

Comment:
Shouldn't we consider `already exists` scenarios ? If the provided UID already exist, user wont be added

Reply:
If so we can do a exit code check and error out

Reply:
Can this be checked ? @sinhaashish

Reply:
@Praveenrajmani : As discussed we are not considering this.